cob_bringup/drivers/usb_camera_node.launch

```diff
@@ -6,11 +6,11 @@
 
 	<node name="$(arg camera_name)" pkg="usb_cam" type="usb_cam_node" output="screen">
 		<param name="video_device" value="/dev/video0"/>
-		<param name="image_width" value="2048"/>
-		<param name="image_height" value="1536"/>
+		<param name="image_width" value="1280"/>
+		<param name="image_height" value="960"/>
 		<param name="pixel_format" value="yuyv"/>
 		<param name="camera_frame_id" value="$(arg camera_name)_link"/>
-		<param name="framerate" value="3.5"/>   <!-- at 2048x1536 3.6hz is maximum on hw -->
+		<param name="framerate" value="10"/>   <!-- at 2048x1536 3.6hz is maximum on hw -->
```

Comment:
do we achieve this rate on the robot?
maybe it makes sense to move these parameters into a yaml file within `cob_hardware_config` as they are camera-specific and this is a general usb_camera_node launch file...

also the hz_monitor.yaml needs to be adjusted to the updated framerate!